zanshinsdk/test_client.py

```diff
@@ -2193,6 +2193,14 @@ def test_validate_uuid_invalid(self):
             zanshinsdk.client.validate_uuid(_uuid)
         except Exception as e:
             self.assertEqual(str(e), f"{repr(_uuid)} is not a valid UUID")
+            
+    def test_validate_uuid_input(self):
+        with self.assertRaises(TypeError):
+            zanshinsdk.validate_uuid(1)
+            zanshinsdk.validate_uuid(None)
```

Comment:
If I understand this code correctly, this line of code will never run and this the None part will not be tested.

I *think* each call to `validate_uuid` needs to be inside its own `with self.assertRaises` block.